tests/PHPUnit/System/expected/test_TwoVisitors_twoWebsites_differentDays_scheduled_report_in_csv__ScheduledReports.generateReport_month.original.csv

```diff
@@ -313,19 +313,15 @@ label,nb_visits,nb_actions,conversion_rate,nb_actions_per_visit,avg_time_on_site
 Unknown,8,40,0%,5,00:15:01,0%
 Desktop,3,3,0%,1,00:00:00,100%
 
-Visitor Browser
-label,nb_visits,nb_actions,conversion_rate,nb_actions_per_visit,avg_time_on_site,bounce_rate
-Unknown,8,40,0%,5,00:15:01,0%
-Firefox,2,2,0%,1,00:00:00,100%
-Opera,1,1,0%,1,00:00:00,100%
-
 Device brand
 label,nb_visits,nb_actions,conversion_rate,nb_actions_per_visit,avg_time_on_site,bounce_rate
 Unknown,11,43,0%,3.9,00:10:55,27%
 
-Device model
+Visitor Browser
```

Comment:
Visitor Browser and Device model both have the same report order id. Therefore it switches sometimes! This is an issue that I do not want to fix here.